calculator.rb

```diff
@@ -0,0 +1,58 @@
+# prints question
+puts "Ready to do some math?"
+# deletes any spaces after the answer provided by the user
+answer = gets.chomp
+
+# ".downcase" takes user input makes it all lowercase
+#  this accounts for any uppercase/lowercase form the word "yes" or "no"
+# depending on user input, if one of the following is true it will prin the line below
+if answer.downcase == "yes"
+  puts "Alrighty then!"
+elsif answer.downcase =="no"
+# "\" escapes quotations, allowing for single quote within the string
+  puts "Whelp, don\'t care!"
```

Comment:
It's good that you're thinking about stuff like this. In this case, you don't need to explicitly escape the single quote, as the string is defined using double quotes. 

Here are some examples of when you'd need the explicit escapes for quotes:
Double quotes: `"hi \"mom\""`
Single quotes: `'hi \'dad\''`